docs/translations.md

```diff
@@ -28,7 +28,7 @@ Placeholders shown in curly brackets `{}` are [translation arguments](https://fo
 3. Don't translate or change proper nouns like `Home Assistant`, `Supervisor` or `Hue`.
 4. For a region specific translation, keys that will be the same as the base translation should be filled with `[VOID]`. These will be replaced during our translation build process.
 5. Translations under the `state_badge` keys will be used for the notification badge display. These translations should be short enough to fit in the badge label without overflowing. This can be tested in the Home Assistant UI either by editing the label text with your browsers development tools, or by using the States tab of developer tools in the Home Assistant UI. In the UI, enter a new entity ID (`device_tracker.test`), and enter the text you want to test in state.
-6. If text will be duplicated across different translation keys, make use of the Lokalise key reference feature where possible. The base translation provides examples of this underneath the `states` translations. Please see the [Lokalise key referencing](https://docs.lokalise.com/articles/1400528-key-referencing) documentation for more details.
+6. Do not use translation keys to build strings, as the string you reuse may be changed, and the places it is used may not be reviewed. If the default (English) string is a translation key, always use this key in the translation as well.
```

Comment:
I think the original text is correct, and the new one isn't.

We want to de-duplicate across different translation keys and use the reference as much as possible.

../Frenck